docs/sites.yml

```diff
@@ -8022,3 +8022,11 @@
     - Blog
     - Education
   featured: false
+- title: asyncbits
+ built_by: HariRVA
+  built_by_url: "https://github.com/gatsbyjs/gatsby/edit/master/docs/sites.yml"
```

Comment:
This needs to be a url for the author, for HariRVA in this case (GitHub profile url is fine too)